Zend/tests/enum/keyword-whitespace.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Enum keyword can be followed by arbitrary whitespaces
+--FILE--
+<?php
+
+enum A {}
+enum B  {}
+enum C	{}
+enum D{}
+enum E	 {}
+enum F
+{}
+enum G
+ {}
```

Comment:
The interesting whitespace is the one between `enum` and the name, not the name and the braces.

Reply:
Lol, complete brainnl fart.

Reply:
This never happened :no_mouth: